setup.py

```diff
@@ -78,7 +78,7 @@
         "License :: OSI Approved :: MIT License",
     ],
     keywords="api graphql protocol rest",
-    packages=find_packages(exclude=["tests"]),
+    packages=find_packages(exclude=["tests", "tests*"]),
```

Comment:
The correct fix should be:
```python
packages=find_packages(include=["graphql_server*"])
```
So everything outside this folder will be ignored, see this [comment](https://github.com/pypa/setuptools/issues/1571#issuecomment-435604162).